index.js

```diff
@@ -88,7 +88,10 @@ class ServerlessDynamodbLocal {
                             convertEmptyValues: {
                                 shortcut: "e",
                                 usage: "Set to true if you would like the document client to convert empty values (0-length strings, binary buffers, and sets) to be converted to NULL types when persisting to DynamoDB.",
-                            }
+                            },
+                            userStop: {
```

Comment:
Shall we make the userStop more meaning full? e.g; stopDbAfterSeeding or feel free to propose a better name. Also, it is important to add a short cut command.